config/migrations/2024/20241129102606-fix-capitilisation-names.sparql

```diff
@@ -0,0 +1,235 @@
+    ( <http://data.lblod.info/id/personen/bafdafda-76ec-441c-898a-99df9559e903> "Degryse" )
+    ( <http://data.lblod.info/id/personen/bdbf9824-f326-4a5a-864a-0f3bd5b74cb6> "Crabbe" )
+    ( <http://data.lblod.info/id/personen/befb8f33670d9bfaf0a8de77d222c6fab1205bf2cabc503595795f1da2ed2642> "Guido" )
+    ( <http://data.lblod.info/id/personen/c9694551-450a-4498-adc5-2c83273fa1ff> "Lesage" )
+    ( <http://data.lblod.info/id/personen/c9b235cf-1d33-4daf-a746-a33dc75da86e> "Lanszweert" )
+    ( <http://data.lblod.info/id/personen/cc23f9bea1af812265d0f2a53b67439add9986ae1133d1e2a54b189502a9ec7f> "Verween" )
+    ( <http://data.lblod.info/id/personen/cbad6f1552a9264ea5cbbf77431e687e828bc2d275082720db242d75282f5e50> "Malyster" )
+    ( <http://data.lblod.info/id/personen/cb26b3251613a9c3ecfe6827fc6b584c4af393ed212deafbdc81d44c037ca540> "Noynaert" )
+    ( <http://data.lblod.info/id/personen/d006b58cb0a71c73cc5529e61f678c8f87e1d77dbd62809a7b7fbef36b16c2e8> "Breemersch" )
+    ( <http://data.lblod.info/id/personen/dafe89eb-1afb-4818-b7be-e3e787cf3b53> "Dolfen" )
+    ( <http://data.lblod.info/id/personen/dadd5a55c1707e46f8dc9728223aabcc2a0c927168f37d89f456a154b0719bc4> "Raskin" )
+    ( <http://data.lblod.info/id/personen/dc5211b6401b87e6c8c5de4ee5762e0a62c468d8a567a1cd81f30bed3b57b1a3> "Van Leuven" )
+    ( <http://data.lblod.info/id/personen/e487f22efe24ea3646f17ec8bbe866559ba469803d959602bb2361f762c762d0> "Verdurme" )
+    ( <http://data.lblod.info/id/personen/e88188fb-c29a-4fa7-b7bd-865fbf883195> "De Coster" )
+    ( <http://data.lblod.info/id/personen/eade770a433a884e1a8a07775c51c4fa2a281af1e3236cfbbe8c8af4caa73b2f> "O" )
+    ( <http://data.lblod.info/id/personen/edc5a623c3d16a5ed60c2fe33436394a6ae590ce7a4761258c79365606734cf4> "O" )
+    ( <http://data.lblod.info/id/personen/eee2ad0169dcbe583a13d77dcfd2d4aa95d3285f9d304b42503a3623b7160ccc> "De Bock" )
+    ( <http://data.lblod.info/id/personen/f3d9246c-6f29-48a8-87f5-c120beb9b2aa> "Ryckman" )
+    ( <http://data.lblod.info/id/personen/f5064b9dadf824c50fe0bfe183450c0be3dae51ca1d534dcea609116b06cf58c> "De Raeve" )
+    ( <http://data.lblod.info/id/personen/f6b389bb57918b6e14a5f469f8825594d9a258513e8462044fcc09efe1d01973> "Vanaudenhove" )
+    ( <http://data.lblod.info/id/personen/fec5999b232cf9e45f683fcf0182fc7763a761924deb800d9bba387aa75ca1cf> "Schollaert" )
+    ( <http://data.lblod.info/id/personen/fdc54cc7-2969-438c-8f1a-a6dede9405c3> "Moerman" )
+    ( <http://data.lblod.info/id/personen/fef09bf23d39c04153ffe155a99775b534a2c899ca7904fde550e4e60ac90f51> "Beek" )
+    ( <http://data.lblod.info/id/personen/bcaccdda-3ec4-418b-a358-8624d6e5b0a3> "O" )
+  }
+  GRAPH ?g {
+    ?s foaf:familyName ?old.
+    OPTIONAL {
+      ?s dct:modified ?oldModified.
+    }
+  }
+  BIND(NOW() AS ?modified)
+  ?g ext:ownedBy ?eenheid.
+};
+
+#Switch first name - last name Guido Van Cutsem
+PREFIX person: <http://www.w3.org/ns/person#>
```

Comment:
i would just delete what was there and add the hard strings guido and van cutsem.
feels more safe even if this would also work

Reply:
probably best indeed, because this also expects a certain order ...